build/lib/prepare_build_container_user.sh

```diff
@@ -31,3 +31,4 @@ groupadd --gid "$GROUP_ID" matchinguser
 useradd  --no-create-home --uid "$USER_ID" --gid "$GROUP_ID" matchinguser
 mkdir -p /home/matchinguser
 chown -R matchinguser:matchinguser /home/matchinguser
+echo "matchinguser ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
```

Comment:
this is to make sudo work when running the builder base container on a linux host, we onyl need this when yum installing, which have not done before (except for Ahree's in progress dhcp build)